src/OneSignal.ts

```diff
@@ -306,7 +306,9 @@ export default class OneSignal {
         MainHelper.checkAndTriggerNotificationPermissionChanged();
       });
 
-      await InitHelper.initSaveState(document.title);
+      await InitHelper.initSaveState(
+        OneSignal.config && OneSignal.config.siteName ? OneSignal.config.siteName : document.title
```

Comment:
After this change it isn't clear that is setting. We could assign the ternary result to a var that is descriptive such as `defaultNotificationTitle`.
```typescript
let defaultNotificationTitle = OneSignal.config && OneSignal.config.siteName ? OneSignal.config.siteName : document.title;
await InitHelper.initSaveState(defaultNotificationTitle);
```